Sources/tart/Commands/Get.swift

```diff
@@ -0,0 +1,31 @@
+import ArgumentParser
+import Foundation
+
+struct Get: AsyncParsableCommand {
+  static var configuration = CommandConfiguration(commandName: "get", abstract: "Get a VM's configuration")
+
+  @Argument(help: "VM name")
+  var name: String
+
+  func run() async throws {
+    do {
+      let vmDir = try VMStorageLocal().open(name)
+      let vmConfig = try VMConfig(fromURL: vmDir.configURL)
+      let diskSize = try vmDir.sizeBytes() / 1024 / 1024 / 1000
+
+      print("cpu\tmemory\tdisk\tdisplay")
```

Comment:
Since this is a human-readable output, what do you think about capitalizing the row names, to be consistent with the `tart list`'s output?

```suggestion
      print("CPU\tMemory\tDisk\tDisplay")
```

Reply:
I think that's a good idea